README.md

```diff
@@ -80,29 +80,71 @@ Think of WebUI like a WebView controller, but instead of embedding the WebView c
 
 ## Documentation
 
- - [Online Documentation - C](https://webui.me/docs/#/c_api)
- - [Online Documentation - C++](https://webui.me/docs/#/cpp_api)
+- [Online Documentation - C](https://webui.me/docs/#/c_api)
+- [Online Documentation - C++](https://webui.me/docs/#/cpp_api)
 
 ## Build
 
- - [Windows](https://github.com/webui-dev/webui/tree/main/build/Windows)
- - [Linux](https://github.com/webui-dev/webui/tree/main/build/Linux)
- - [macOS](https://github.com/webui-dev/webui/tree/main/build/macOS)
-
+Clone the repository
+```
+git clone https://github.com/webui-dev/webui.git
+```
+
+Build the library
+
```

Comment:
The `## Build` section is updated to conform with the upcoming changes to the building process, and also includes an info about building the client. While writing this comment I realize that the client dependencies should be added as well. I will update the last commit to include them.

To see the GitHub rendered version of the Readme please check:
https://github.com/ttytm/webui/tree/build-single-makefile-in-root

Any critique is welcome.

Reply:
Another way to add build instructions would be to enable the wiki for the repo and move the build section from the Readme to the wiki and refer to it with a link. E.g., like neovim does: https://github.com/neovim/neovim#install-from-source